bower.json

```diff
@@ -2,7 +2,7 @@
   "name": "ember-local-storage",
   "dependencies": {
     "blob-polyfill": "~1.0.20150320",
-    "ember": "~2.10.0",
+    "ember": "components/ember#lts-2-4",
```

Comment:
Ah shoot, sloppy of me -- that's a merge error, didn't mean to change anything here. Sorry, fixing...